modules/Bio/EnsEMBL/Variation/DBSQL/VariationFeatureAdaptor.pm

```diff
@@ -1844,24 +1844,33 @@ sub fetch_by_hgvs_notation {
     #Get the Transcript object to convert coordinates
     my $transcript_adaptor = $user_transcript_adaptor || $self->db()->dnadb()->get_TranscriptAdaptor();
     my $transcript = $transcript_adaptor->fetch_by_translation_stable_id($reference);
-    
+
     my @transcripts;
 
-    #try and fetch via gene
+    # support some malformed HGVS
     if(!defined($transcript)) {
-      push @transcripts, @{$self->_get_gene_transcripts($transcript_adaptor, $reference, $multiple_ok)};
+      # Seeing transcripts erroneously submitted with p. changes
+      if($reference =~ /ENST|NM/){
+         push @transcripts, $transcript_adaptor->fetch_by_stable_id($reference);
+      }
+      else{
+        #try and fetch via gene
+        $multiple_ok = 1;  ## for VEP - return all possible here but report 1st with matching ref base
```

Comment:
$multiple_ok is set on calling fetch_by_hgvs_notation. Variant Recoder sets it to 1 by default and VEP to 0. I thought it would be clearer to obviously override it here, but maybe not?